token/cli/src/config.rs

```diff
@@ -326,13 +330,16 @@ impl<'a> Config<'a> {
         wallet_manager: &mut Option<Arc<RemoteWalletManager>>,
         token: Option<Pubkey>,
     ) -> Result<Pubkey, Error> {
-        if let Some(address) = pubkey_of_signer(arg_matches, override_name, wallet_manager).unwrap()
-        {
+        let address = match pubkey_of_signer(arg_matches, override_name, wallet_manager) {
+            Ok(address) => address,
+            Err(e) => return Err(e.to_string().into()),
+        };
```

Comment:
nit: how about mapping the error?
```suggestion
        let address = pubkey_of_signer(arg_matches, override_name, wallet_manager).map_err(|e| e.to_string().into())?;
```